script.js

```diff
@@ -1,4 +1,145 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+
+
+// There are 2 game modes
+var gameModeDiceRoll = 'gameModeDiceRoll';
+var gameModeDiceOrder = 'gameModeDiceOrder';
+
+//for each roll of two independent dice
+var rollOfDice = [];
+
+//To keep track of each player playerCombineNumber
+var playerCombineNumber;
+var playerCombineNumber1 = [];
+var playerCombineNumber2 = [];
+
+//Start game with Player 1
+var currPlayer = 1;
+
+//variable for winner
+var winner = "";
+
+//Start game with gameModeDiceRoll
+var gameMode = gameModeDiceRoll;
+
+var main = function (input) { 
+  if (input.toLowerCase() == "r"){
+    var finalScore1 = player1Result();
+    var finalScore2 = player2Result();
+
+    if(finalScore1 > finalScore2){
+      winner = 1;
+    }
+    else if (finalScore2 > finalScore1){
+      winner = 2;
+    }
+    return `Winner is Player ${winner} <br> Player 1 score is ${finalScore1} <br> Player 2 score is ${finalScore2}`;
+  };
+
+  if (gameMode == gameModeDiceRoll){
+      rollOfDice = getDiceRolls();
```

Comment:
Great job using rollOfDice for both players and then emptying out the array at the end of each player's turn!